.github/workflows/validate-status-schema.yml

```diff
@@ -0,0 +1,98 @@
+# Validate status.json schema
+# Runs on a schedule every Friday to ensure the JSON structure matches expectations
+# Alerts if schema has changed so updates can be made to dependent scripts
+
+name: Validate Status JSON Schema
+
+on:
+  workflow_dispatch:
+    inputs:
+      statusFileUrl:
+        description: 'Optional: Custom status.json URL to validate (for testing)'
+        required: false
+        type: string
+
+  pull_request:
+    paths:
+      - .github/workflows/validate-status-schema.yml
+      - .github/workflows/validate-status-schema.ps1
+      - .github/workflows/library.ps1
```

Comment:
We do not need to run on every single change in the library file, so lets remove that
```suggestion
```